tools/serverpod_cli/lib/src/logger/loggers/std_out_logger.dart

```diff
@@ -98,54 +127,75 @@ class StdOutLogger extends Logger {
   void _log(
     String message,
     LogLevel logLevel,
+    bool newParagraph,
     LogStyle style, {
     String prefix = '',
   }) {
     if (message == '') return;
     if (!shouldLog(logLevel)) return;
 
-    if (style is BoxLogStyle) {
+    if (style is BoxLog) {
       message = _formatAsBox(
+        wrapColumn: wrapTextColumn ?? 100,
         message: message,
         title: style.title,
       );
-    } else if (style is TextLogStyle) {
+    } else if (style is TextLog) {
       if (wrapTextColumn != null && style.wordWrap) {
         message = _wrapText(message, wrapTextColumn!);
       }
 
       switch (style.type) {
-        case AbstractStyleType.command:
-          message = '  \$ $message';
+        case TextLogType.command:
+          message = '   ${AnsiStyle.yellow.wrap('\$ $message')}';
```

Comment:
I think it would be nicer to just highlight the `$` sign here. Perhaps use the turquoise color that was used for "Creating Serverpod project ..." message?

Extra cool would be to syntax highlight the command, but that would probably be too much work for now.